docs/install.md

```diff
@@ -119,15 +119,11 @@ NXF_VER=24.06.0-edge nextflow info
 
 ## Standalone distribution
 
-Nextflow has a set of {ref}`core plugins <plugins-core>` which are downloaded at runtime by default. There is also a standalone distribution (i.e. the `all` distribution) which comes pre-packaged with all core plugins. This distribution is mainly useful for offline environments.
+Nextflow has a set of {ref}`core plugins <plugins-core>` which are downloaded at runtime by default. There is also a standalone distribution (i.e. the `dist` distribution) which comes pre-packaged with all core plugins. This distribution is mainly useful for offline environments.
```

Comment:
The `dist` doesn't include all core plugins anymore, should remove that part and/or specify which plugins are included

Reply:
Good point, updated.